Sources/FluentUI_iOS/Components/Notification/MSFNotification.swift

```diff
@@ -61,11 +79,13 @@ import UIKit
     ///   - anchorView: The view used as the bottom anchor for presentation
     ///   (notification view is always presented up from the anchor). When no anchor view is provided the
     ///   bottom anchor of the container's safe area is used.
+    ///  - layoutGuide: The layout guide used to present the toast inside of when no `anchorView` is provided
```

Comment:
Can we describe "how" the layout guide is used? Which part of the notification is aligned with the layout guide? And probably worth explaining what happens if both an `anchorView` and a `layoutGuide` are provided